### ISSUE TITLE
Please add MOSH support

### ISSUE THREAD

**Issue by readyblue** [connectbot/connectbot#123]
please add MOSH support from: https://mosh.mit.edu/
Thanks

**Comment by robert7k** [comment]
There used to be a fork of ConnectBot offering MOSH support:
http://dan.drown.org/android/mosh/
https://github.com/ddrown/irssiconnectbot

The project is not maintained anymore, but maybe the code can be merged.

**Comment by kruton** [comment]
I like MOSH as well, but the license appears to be a problem.

**Comment by robert7k** [comment]
IrssiConnectBot included MOSH as a binary, see here:
https://github.com/ddrown/irssiconnectbot/tree/master/binary-packages

Shouldn't avoid that the license problem?

**Comment by readyblue** [comment]
For those of you consider this must-have feature,
Try https://termux.com/ - Termux (app) , licenced under GPLv3
To get it work, you need to install it by:
`apt install mosh`
No root privileges is needed for that command, but you may need to update repository first by:
`apt update`

**Comment by ciechomke** [comment]
Why license is a problem? Mosh is licensed under GPLv3.

**Comment by ShalokShalom** [comment]
THIS

Mosh brings obvious benefits and since there is already code for it..

**Comment by hupfdule** [comment]
Mosh is also supported by juicessh. Would be great if connectbot supports it as well.

**Comment by paxperscientiam** [comment]
@hupfdule you're right, but it seems author is AWOL. I've been trying to get a hold of him for months regarding payment issue. No response.
